iterable-extension/src/test/java/com/mparticle/ext/iterable/IterableExtensionTest.java

```diff
@@ -162,6 +162,19 @@ public void testUpdateUser() throws Exception {
         Map<String, String> userAttributes = new HashMap<String, String>();
         userAttributes.put("some attribute key", "some attribute value");
         request.setUserAttributes(userAttributes);
+        Map<String, List<String>> userAttributeList = new HashMap<>();
+        String jsonString =
+                "{" +
+                    "\"country\": \"France\"," +
+                    "\"depart\": \"2022-05-01\"," +
+                    "\"nights\": 7," +
+                    "\"adults\": 2," +
+                    "\"children:\": 0," +
+                    "\"nearestAirports\": [\"LGW \", \"MAN\", \"BHX\", \"GLA\"]," +
+                    "\"properties\": [13404, 13401, 13152, 13145]" +
+                "}";
+        userAttributeList.put("abandonedBrowse", Arrays.asList("Rome", "San Juan", jsonString));
```

Comment:
Not important but Iterable doesn't allow heterogenous lists like this. It might be good to stick `jsonString` in a different list in case anyone in the future uses this test as a guide to using list attributes.